charts/nextcloud/values.yaml

```
@@ -3,7 +3,9 @@
 ##
 image:
   repository: nextcloud
-  # tag: 24.0.3-apache
+  flavor: apache
+  # -- tag: 24.0.3-apache
```

```suggestion
  # tag: 24.0.3-apache
```
Or do these serve any purpose?